### DISCUSSION TITLE
$LIN/tools

### DISCUSSION THREAD

**Discussion by avmaisak** [Linux4Yourself/book#553] [Ideas]
может вынести в переменную окружения? Что то вроде $LIN_TOOLSDIR

**Comment by makpif** [comment]
что это даст?

**Reply by avmaisak** [reply]
использование единой переменной для хранения пути, при необходимости смены пути достаточно поменять в одном месте.

**Comment by Linuxoid85** [comment]
Я только за. Но "необходимость смены пути", думаю, редко возникает. Но вот писать вечные `$LIN/tools`, когда можно сократить, скажем, до `$LIN_TLDR` (или чего-то подобного) - это отлично.

**Reply by avmaisak** [reply]
Да я про это.

**Reply by makpif** [reply]
$LIN/tools - 10 символов
$LIN_TLDR - 9 символов

Прям очень большой выигрыш. Ну а $LIN_TOOLSDIR - 13 символов вообще

я против, только добавляение путаницы. 

Выводить в переменныую следует только то что может потребоватся изменить ( путь к разделу например)
